src/components/Form/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import ManOutlinedIcon from "@mui/icons-material/ManOutlined";
+import WomanOutlinedIcon from "@mui/icons-material/WomanOutlined";
+const UserForm = () => {
+  const [name, setName] = useState();
```

Comment:
Can you use one state for all the user details?

Reply:
## one useState is used for all userDetails in modified code